src/api/procedures/consumeAddMultiSigSignerAuthorization.ts

```diff
@@ -100,7 +98,7 @@ export async function getAuthorization(
   } else {
     calledByTarget = !!identity?.isEqual(target);
     hasRoles = calledByTarget;
-    transactions = [TxTags.multiSig.AcceptMultisigSignerAsIdentity];
+    const transactions = [TxTags.multiSig.AcceptMultisigSignerAsIdentity];
     permissions = { transactions };
```

Comment:
```suggestion
    permissions = { transactions:  [TxTags.multiSig.AcceptMultisigSignerAsIdentity] };
```

I didn't realize it was being used later. This should get rid of the code smell